bin/merge.sh

```diff
@@ -183,20 +202,95 @@ rejection_diff() {
 
 }
 
+need_40s_40x-dv_merge(){
```

Comment:
I'm wondering if this needs to be so complicated.

Would something like this be equivalent?
```
theirs_head_commit=$(git rev-parse theirs_branch_name)
theirs_is_parent=$(git branch --contains $theirs_head_commit)
if [ $theirs_is_parent ]; then
    echo "No need to merge"
    exit 0
fi
```